modules/031-linstor/docs/CONFIGURATION.md

```diff
@@ -98,3 +98,249 @@ In case your application does not support high availability and runs in a single
 To solve the problem, specify the label `linstor.csi.linbit.com/on-storage-lost: remove` in the Pod description. The linstor module will automatically remove such Pods from the node where the problem occurred, allowing Kubernetes to restart the application on another node.
 
 [Example...](usage.html#application-reschedule-in-case-of-node-problem-storage-based-fencing)
+
+## Backup on S3 Storage
+
+Data backup is implemented using [volume snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/). Snapshots are supported by the [snapshot-controller](../045-snapshot-controller/) module, which is enabled automatically for supported CSI drivers in Kubernetes clusters versions 1.20 and higher.
+
+> It works with [Amazon S3](https://aws.amazon.com/s3/) and [MinIO](https://min.io). There may be problems when working with other services.  
+>
```

Comment:
```suggestion
```

fixed in [v1.19.0](https://github.com/LINBIT/linstor-server/releases/tag/v1.19.0)